test/stripe/ephemeral_key_test.rb

```diff
@@ -0,0 +1,103 @@
+require File.expand_path('../../test_helper', __FILE__)
+
+module Stripe
+  class EphemeralKeyTest < Test::Unit::TestCase
+    FIXTURE = {
+      'id': 'ephkey_123',
```

Comment:
We're still supporting Ruby < 2.2, and older versions don't like colon syntax for use with string keys in hashes, so you'll need to switch these (and hashes in a few other places) to `=>`.

You should be able to find some failures in CI for details.